index.html

```diff
@@ -0,0 +1,310 @@
+                                <li class="skill-card-tech-tag rails"><p class="skill-card-tech-tag-text">Ruby on rails</p></li>
+                                <li class="skill-card-tech-tag rspec"><p class="skill-card-tech-tag-text">RSpec</p></li>
+                            </div>
+                            <div class="skill-card-tech">
+                                <li class="skill-card-tech-tag capybara"><p class="skill-card-tech-tag-text">Capybara</p></li>
+                                <li class="skill-card-tech-tag selenium"><p class="skill-card-tech-tag-text">Selenium</p></li>
+                            </div>
+                        </ul>
+                    </div>
+                </section>
+                <!-- skills -->
+                <section class="skill-card">
+                    <!-- icon -->
+                    <div class="skill-card-icon">
+                        <div class="skill-card-icon-container"><img class="skill-card-icon-img" src="img/skill/skills-img.svg" alt=""></div>
+                    </div>
+                    <!-- title and tags -->
+                    <div  class="skill-card-content">
+                        <h3 class="skill-card-title">Skills</h3>
+                        <ul class="skill-card-rows">
+                            <div class="skill-card-tech">
+                                <li class="skill-card-tech-tag codekit"><p class="skill-card-tech-tag-text">Codekit</p></li>
+                                <li class="skill-card-tech-tag github"><p class="skill-card-tech-tag-text">GitHub</p></li>
+                                <li class="skill-card-tech-tag codepen"><p class="skill-card-tech-tag-text">Codepen</p></li>
+                                <li class="skill-card-tech-tag gitlab"><p class="skill-card-tech-tag-text">Gitlab</p></li>
+                            </div>
+                            <div class="skill-card-tech">
+                                <li class="skill-card-tech-tag terminal"><p class="skill-card-tech-tag-text">Terminal</p></li>
+                            </div>
+                        </ul>
+                    </div>
+                </section>
+            </div>
+        </section>
+        
+        <!-- Form Section -->
+        <section id="form">
+            <!-- text -->
+            <div id="form-text">
+                <p id="form-text-item">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+            </div>
+            <!-- form -->
+            <form action="https://formspree.io/f/xyyakzgb" method="POST" id="form-content">
+                <div class="form-row" id="first-form-row">
+                    <input type="text" name="FirstName" max-length="30" name="first-name" placeholder="First Name" class="form-row-column" required>
+                    <input type="text" name="SecondName" max-length="30" name="last-name" placeholder="Last Name" class="form-row-column" required>
```

Comment:
- [ ] Great job implementing the desktop version of the portfolio, As mentioned in the project requirements, I think it will be better if you stay with one input for full name just like in the mobile version since it will be a better user experience 
![image](https://user-images.githubusercontent.com/97521557/220352860-61f76d43-eefc-405a-aced-8f3ff82d0651.png)